### PR TITLE
Flesh out the bundle api.

### PR DESCRIPTION
**Pull request by ttung**
1) Add `list_versions`, which just returns a list of versions.
2) Given a bundle UUID+version, include a list of files (i.e., `file_version`)

### CODE REVIEW

File: dss-api.yml [hunk 1]
```diff
@@ -208,6 +230,24 @@ definitions:
...
+        description: Bundle unique ID
+      bundle_timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of bundle creation measured in milliseconds since epoch
+      contents:
+        type: array
+        items:
+          $ref: "#/definitions/file_version"
+      writer_uid:
```
Comment (kislyuk) [inline dss-api.yml]: Should we call this "created_by" or "creator_uid"?

File: dss-api.yml [hunk 2]
```diff
@@ -208,6 +230,24 @@ definitions:
...
+  bundle_version:
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: Bundle unique ID
+      bundle_timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of bundle creation measured in milliseconds since epoch
```
Comment (kislyuk) [inline dss-api.yml]: Same question as elsewhere, timestamp format - iso or epoch?

File: dss-api.yml (hunk 1 above)
Comment (ttung) [inline dss-api.yml]: I don't like `created_by`, since there's ambiguity about the data type (name?  uid?).  I'll change it to `creator_uid`.

**Comment by ttung** [comment]
I changed this but I don't know how to re-request review... :(

**Comment by kislyuk** [comment]
LGTM thanks!

**Comment by kislyuk** [comment]
Did you mean to merge this into `ttung_rename` or master? I can re-merge into master if the latter.

**Comment by ttung** [comment]
It was intended for master, but in order to show stacked diffs correctly, I had to use ttung_rename as the base.  Github is not great at stacked diffs. :(

**Comment by ttung** [comment]
I fixed it and manually pushed.  Thanks for flagging it, @kislyuk!

**Pull request merged**